samples/vars/callhome_vars.yml

```diff
@@ -0,0 +1,22 @@
+---
+#
+# saples/vars/callhome_vars.yml
+#
+
+# Refer to `man mmcallhome` man pages for a description of these Call Home
+# parameters.
+
+scale_callhome_params:
+  is_enabled: true
+  customer_name: abc
+  customer_email: abc@abc.com
+  customer_id: 12345
+  customer_country: IN
```

Comment:
Should we have a variable written with "  example:  "IN"    
Ansible will read Norway as NO if don't use customer_country: "NO"